test/v1/configuration.go

```diff
@@ -36,6 +36,7 @@ import (
 // that uses the image specified by names.Image.
 func CreateConfiguration(t pkgTest.T, clients *test.Clients, names test.ResourceNames, fopt ...rtesting.ConfigOption) (*v1.Configuration, error) {
 	config := Configuration(names, fopt...)
+	test.AddTestLabel(t, &config.Labels)
```

Comment:
So configuration and revision, etc usually derive their labels from service.
Do we need to (well I can see for config and route if they're top level, rather than created via kservice) set the label on them?

Reply:
I think it's a good idea for consistency to set the label on all resources directly created by the tests.